core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -95,13 +95,7 @@ export class OpportunitiesListComponent {
           this.opportunities = this.opportunities.filter((opportunity) => {
             return opportunityIds.indexOf(opportunity.id) < 0;
           });
-          this.visibleOpportunities = this.opportunities.slice(
-            0, this.OPPORTUNITIES_PAGE_SIZE);
-          this.userIsOnLastPage = this.calculateUserIsOnLastPage(
-            this.opportunities,
-            this.OPPORTUNITIES_PAGE_SIZE,
-            this.activePageNumber,
-            this.more);
+          this.gotoPage(this.activePageNumber);
```

Comment:
Is there any scenario where, when opportunities are removed, the activePageNumber is no longer valid (i.e. it contains zero opportunities)? In such cases should we be sending users to the last available page instead?

Reply:
I get the "show more" case, but what I'm asking about is the "not enough" case -- e.g. suppose we're on page 8, seeing results 71-72, and then opportunities get removed, so there isn't a page 8 any more (since the number of open opportunities drops below 70). In that case, what happens?